### DISCUSSION TITLE
Contribution on the project

### DISCUSSION THREAD

**Discussion by Ressley** [ocornut/imgui#5172] [New Users Build/Link/Run issues ONLY!]
I am a newbie in the OSS community and want to contribute.
Can you give me some advises or task to work on?

**Comment by ocornut** [comment]
Unusually enough my answer would be to discourage random contributions. Dear ImGui has become a complex project. Contributions must come from the angle of solving a real problem that you have and really care about, rather than picking among a random list. Make use of it and you’ll probably stumble on something meaningful to do :)

Other projects may be better structured to welcome new developers but we aren’t unfortunately. Many (not all) of the easy problems are already solved. The remaining ones are often not trivial. But there’s still plenty to do.
